mlos_bench/mlos_bench/launcher.py

```diff
@@ -180,7 +181,11 @@ def _parse_args(parser: argparse.ArgumentParser) -> Tuple[argparse.Namespace, Li
             dest='teardown', action='store_false',
             help='Disable teardown of the environment after the benchmark.')
 
-        return parser.parse_known_args()
+        (args, args_rest) = parser.parse_known_args()
+        args.config_path = list(chain(*args.config_path))
+        args.tunable_values = list(chain(*args.tunable_values))
+        args.globals = list(chain(*args.globals))
+        return (args, args_rest)
```

Comment:
```suggestion
        return parser.parse_known_args()
```
`action='extend'` should flatten the lists automatically